drawer/draw.py

```diff
@@ -24,16 +24,13 @@ def draw(self):
         return items
 
     def __shuffle_pack(self, n_players):
-        shuffled = Random(self.__generate_daily_seed()).sample(self.__options, k=n_players)
```

Comment:
I would leave it as internal random, cause when we execute some kind of shuffle outside, we'll get wrong seed and repeated value